.github/workflows/track_todo.yml

```diff
@@ -0,0 +1,17 @@
+name: Track TODOs
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
Lets pin `actions/checkout` to a commit version, we use `actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f`.